backup_and_restore/application_backup_and_restore/installing/installing-oadp-ibm-cloud.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oadp-ibm-cloud"]
+= Configuring the OpenShift API for Data Protection with {ibm-cloud-title}
+include::_attributes/common-attributes.adoc[]
+:context: installing-oadp-ibm-cloud
+:installing-oadp-ibm-cloud:
+:credentials: cloud-credentials
+
+
+toc::[]
+
+You install the {oadp-first} Operator on an {ibm-cloud-title} cluster to back up and restore applications on the cluster. You configure {ibm-cloud-object-storage} to store the backups.
```

Comment:
```suggestion
You can install the {oadp-first} Operator on an {ibm-cloud-title} cluster to back up and restore applications on the cluster. You can configure {ibm-cloud-object-storage} to store the backups.
```

Personally, I would suggest using `can` wherever possible.

Reply:
Actually, I have been recommended by other OCP reviewers to avoid usage of 'can'. I understand it makes sense here and it is a good suggestion. I will hold making this edit for the sake of consistency. Thanks.